rust/feature-flags/src/flags/flag_service.rs

```diff
@@ -0,0 +1,388 @@
+use common_metrics::inc;
```

Comment:
none of this is new code, it was just pulled out of `flag_request` along with the tests.  Should work fine.